### DISCUSSION TITLE
osu!lazer doesn't open on OSX after changing from Metal to OpenGL

### DISCUSSION THREAD

**Discussion by themii304** [ppy/osu#23589] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I am on OSX 12.6.5 Monterey / Intel chip

After changing the rendering settings to OpenGL, my game just opens and stays in the taskbar. I have tried deleting the framework file and restarting/shutting down my Mac and reinstalling the latest release of lazer.

All help appreciated

### Version

Latest Release

### Logs

I couldn't find any log folder/files in the game files

**Comment by peppy** [comment]
Deleting `framework.ini` from the correct location should be enough to resolve this. Make sure to restart your PC in case osu! happens to be stuck running in the background

**Reply by themii304** [reply]
Not sure where to find this, unless its the same thing as osu.framework.dll?
The only way I know to get into the files is by right clicking sou! and selecting show package contents, not sure where else I would find any game files.

**Reply by peppy** [reply]
`~/Library/Application Support/osu` probably.

**Reply by themii304** [reply]
No luck, here's what happens when I search for the game 

https://github.com/ppy/osu/assets/133903853/be8da64d-62a2-4357-800e-15df59c98200

**Reply by Theighlin** [reply]
Deleted because dumb (i forgot you couldn't open the game). I think if you installed the game a long time ago it might be a different path, see https://github.com/ppy/osu/discussions/19093

**Reply by peppy** [reply, marked as answer]
Wrong library folder. Please google and learn your OS. It's the one in your home folder.

**Reply by themii304** [reply]
I had to go onto my user and press CMD + SHIFT + . (dot) found the files and it works now thanks!
